tests/specs/cli/otel_basic/context.ts

```diff
@@ -0,0 +1,26 @@
+import { assertEquals } from "@std/assert";
```

Comment:
For future readers: spec test rather than unit test because of `--unstable-otel`